inst/private/python_ipc_driver.m

```diff
@@ -73,5 +73,10 @@
       [A, db] = python_ipc_sysoneline(what, cmd, false, varargin{:});
 
     otherwise
-      error('invalid ipc mechanism')
+      if (strcmp(what, 'reset'))
+        A = true;
+        db = [];
+      else
+        error('invalid ipc mechanism')
```

Comment:
Its ok to move these things to Octave-formating-style when updating existing code.  (well, it should still be matlab-dialect).